internal/docstore/driver/driver.go

```diff
@@ -104,10 +111,21 @@ type Filter struct {
 	Value     interface{} // the value to compare using the operation
 }
 
-// A DocumentIterator is an iterator that can iterator through the results (for Get
-// action), and returns the next page that can be passed into a Query as a
-// starting point.
+// A DocumentIterator iterates through the results (for Get action).
 type DocumentIterator interface {
-	Next(doc Document) error
+
+	// Next tries to get the next item in the query result and decode into Document
+	// with the driver's codec.
+	//
+	// When the driver makes the query request for a page size smaller than the
```

Comment:
I would just delete this paragraph; it's talking about pages, when the interface is not paginated.

Reply:
Done.